certified-connectors/USB4SAP/apidefinition.swagger.json

```diff
@@ -0,0 +1,70 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "USB4SAP",
+    "version": "1.0",
+    "description": "Universal Synch Bridge for SAP: One API for all SAP integration, analytics and UI needs. Simplify modern analytics, apps and automations. Provision SAP data, insights and automation On-demand.",
+    "contact":{
+      "name": "Ecoservity",
+      "url" : "https://www.ecoservity.com/#contact",
+      "email": "hello@ecoservity.com"
+
+    }
+  },
+
+  "host" : "http://35.208.99.120:8000",
+  "paths": {
+    "/sap/opu/odata/ECOS/OBJ2CLOUD_V2_SRV/ET_DatasetSet": {
+      "GET": {
+        "summary": "GET call to extract metadata & data from SAP data objects",
```

Comment:
@ecsadm `&` sign is not allowed here, one of the checks is failing because of it. Please replace it by `and`.
Please see [English strings recommendations](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings) or [connector string guidance](https://docs.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info) for more information about strings.